packages/kbn-datemath/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@kbn/datemath",
+  "version": "5.0.0",
```

Comment:
We shouldn't need to version this. We should only even need to include a license if it's forked from a separate project (which this particular package is).

Reply:
Haha, I'm sooo used to npm complaining that I've used added because of that. But yarn isn't complaining about missing version, so I'll 🔥 it

Reply:
Npm wouldn't complain either because this package is marked as private, which disables pretty much all of the npm integrity checks.

Reply:
Aaand it's a no-go:

```
18:35:14 Installing dependencies in [kibana]:
18:35:14 
18:35:15 $ node ./preinstall_check
18:35:15 [1/5] Validating package.json...
18:35:15 [2/5] Resolving packages...
18:35:15 error Package "@kbn/datemath@" doesn't have a "version".
18:35:15 info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

Reply:
What? Why on earth would it do that? Maintaining versions on these packages doesn't make any sense, and it's definitely going to get out of date really fast. I'd argue it's out of date the moment it is contributed to the project :P

Reply:
Why? I expect because `link:` and `private: true` are edge-cases.

Reply:
Just curious, and this seems like a decent place to ask; why not just pin all the package versions to the Kibana version in the root, ala lerna's default behavior (ie. not independent mode)? They're all technically new packages, so it's not like we'd be rolling back versions or anything. And they're all pretty hard-bound to Kibana anyway...

Reply:
++ I've been thinking about the same thing. We could write some tooling in `yarn kbn` to handle it. It could require exact version match on all `private: true` packages, for example. That way we could still have publishable packages, _if we want to_ (not sure we do want that any more, though, so it might not be necessary at all and we just require exact same version on _every_ package)

Reply:
Yeah, that's what I was thinking too. A `kbn version`, similar to `npm version` (or I assume lerna, I've only used lerna with independent versioning so far) that would just pump core and all the packages with it.